addons/project_todo/wizard/mail_activity_todo_create.py

```diff
@@ -0,0 +1,35 @@
+from odoo import fields, models, _
+
+class MailActivityTodoCreate(models.TransientModel):
+    _name = 'mail.activity.todo.create'
+    _description = 'Create activity and todo at the same time'
+
+    summary = fields.Char('Summary')
```

Comment:
Shouldn't this be required? Also, I don't think the string "Summary" is necessary. :slightly_smiling_face: